packages/api/src/command/medical/patient/create-medical-record.ts

```diff
@@ -74,10 +85,40 @@ export async function getMedicalRecordSummary({
   const pdfIsValid = conversionType === "pdf" && pdfCreatedAt;
   const htmlIsValid = conversionType === "html" && htmlCreatedAt;
 
+  let s3BucketName = bucketName;
+  let s3FileKey;
```

Comment:
No need to declare this here as a variable, could have it as a const inside each block, right? "consts >>> variables" 🤓